addon/components/snippet-plugin/snippet-list-select-rdfa.ts

```diff
@@ -49,8 +49,11 @@ export default class SnippetListSelectRdfaComponent extends Component<Args> {
     );
   }
 
-  saveChanges = (snippetIds: string[]) => {
+  saveChanges = (snippetIds: string[], snippetNames: string[]) => {
     if (this.currentResource) {
+      this.controller.withTransaction((tr) => {
```

Comment:
This removes the name in a separate transaction to the update of the list ids, which puts it in it's own entry in the undo stack. You can see this by modifying the snippet lists assigned to a placeholder, then hitting ctrl-z (you need to de-select the node for some reason, this is most likely an ember-node bug); you should see the same list of names in the node (as after the change) but if you hit 'manage snippet lists' you'll see the original selections from before the change.

I think we need to modify the `updateSnippetIds` command.

Reply:
Didn't think of the undo command, thanks, should be fixed now